keras_core/ops/image_test.py

```diff
@@ -169,24 +227,18 @@ def test_affine_transform(self, interpolation, fill_mode, data_format):
         )
         if data_format == "channels_first":
             x = np.transpose(x, (1, 2, 0))
-        ref_out = tf.raw_ops.ImageProjectiveTransformV3(
-            images=tf.expand_dims(x, axis=0),
-            transforms=tf.cast(tf.expand_dims(transform, axis=0), tf.float32),
-            output_shape=tf.shape(x)[:-1],
-            fill_value=0,
-            interpolation=interpolation.upper(),
-            fill_mode=fill_mode.upper(),
+        coordinates = _compute_affine_transform_coordinates(x, transform)
+        ref_out = scipy.ndimage.map_coordinates(
```

Comment:
Here I replaced `tf.raw_ops.ImageProjectiveTransformV3` with `scipy.ndimage.map_coordinates` because scipy has more fill_mode options to test all backends.